### DISCUSSION TITLE
Select framework features

### DISCUSSION THREAD

**Discussion by jonatandorozco** [laravel/framework#36829] [Q&A]
I would like to know if there is a way to select the features I want to use in order to reduce application size and improve performance?

I know Laravel is a full featured framework but I think can be better if it can be modular

Thanks

**Comment by rodrigopedra** [comment]
You can enable/disable features by commenting out the loaded Service Providers on your app's `./config/app.php` config file:

https://github.com/laravel/laravel/blob/5808129a1f702f973c7c31203d16db2066bd9030/config/app.php#L137-L178

But note some features are inter-dependent. For example, the `MailServiceProvider` needs `ViewServiceProvider` (to render the mailables), `SessionServiceProvider` requires `CookieServiceProvider`, etc.
